tasks/purge-registry.yml

```
@@ -0,0 +1,25 @@
+---
+- name: Copy init script
+  template:
+    src: 'purge-registry.py.j2'
+    dest: '{{ gitea_service_path }}/purge-registry.py'
+    owner: '{{ gitea_app_host_uid }}'
+    group: docker
+    mode: 0755
+
+- name: 'Create timer for launch the clean up of workspace '
+  include_role: name=systemd-timer
+  vars:
+    systemd_timer_name: '{{ gitea_container_purge_timer_name }}'
+    systemd_timer_description: 'Timer to perform clean '
+    systemd_timer_dynamic_user: true
+    systemd_timer_frequency: '{{ gitea_container_purge_timer_frequency }}'
+    systemd_timer_timeout_sec: '{{ gitea_container_purge_timer_timeout }}'
+    systemd_timer_work_dir: '{{ gitea_service_path }}'
+    systemd_timer_requires_extra: 'docker.service'
+    systemd_timer_start_on_creation: false
+    systemd_timer_script_path: '{{ gitea_service_path}}/purge-registry.py'
+    systemd_timer_script_args: '--url http://localhost:{{ gitea_app_cont_web_port }}  --retention {{ gitea_purge_max_retention }}'
```

You can make this more readable using `>-`:
```suggestion
    systemd_timer_script_args: >-
      --url http://localhost:{{ gitea_app_cont_web_port }}
      --retention {{ gitea_purge_max_retention }}
```